gc/startup/omrgcalloc.cpp

```diff
@@ -33,16 +33,6 @@
 omrobjectptr_t
 OMR_GC_AllocateObject(OMR_VMThread * omrVMThread, MM_AllocateInitialization *allocator)
 {
-	MM_EnvironmentBase *env = MM_EnvironmentBase::getEnvironment(omrVMThread);
-
-	/* TODO: Deprecate this -- required for early versions of ruby/omr integration */
-	if (allocator->isGCAllowed() && (NULL == env->getExtensions()->getGlobalCollector())) {
-		/* Lazily create the collector before attempting to allocate. */
-		if (OMR_ERROR_NONE != OMR_GC_InitializeCollector(omrVMThread)) {
-			return NULL;
-		}
-	}
-
```

Comment:
LGTM

Reply:
Can we assert that the global collector is NOT null.